resources/services/ecr/repositories.go

```diff
@@ -241,3 +563,78 @@ func fetchEcrRepositoryImages(ctx context.Context, meta schema.ClientMeta, paren
 	}
 	return nil
 }
+func fetchEcrRepositoryImageScanFindings(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	maxResults := int32(1000)
+	image := parent.Item.(types.ImageDetail)
+	config := ecr.DescribeImageScanFindingsInput{
+		ImageId:        &types.ImageIdentifier{ImageDigest: image.ImageDigest},
+		RepositoryName: image.RepositoryName,
+		MaxResults:     &maxResults,
+	}
+	c := meta.(*client.Client)
+	svc := c.Services().ECR
+	for {
+		output, err := svc.DescribeImageScanFindings(ctx, &config, func(options *ecr.Options) {
+			options.Region = c.Region
+		})
```

Comment:
This is the call I am worried about, Every Repo will have to make at least `1` API call for each image, and each of those extra calls is done sequentially

Reply:
@bbernays @irmatov  I see this PR has been parked for a while now--is this the blocker, and is there an alternative solution we can consider?

Reply:
The only other solution is to make `image scan results` be its own table. In order to do that we would have to duplicate the call to list repositories call, which might be a reasonable trade off...